src/core/scale/baseScale.js

```diff
@@ -77,9 +85,21 @@ export function baseScale(config = {}, parent) {
         },
         get domain() {
             trace();
-            return this.config.domain ? this.config.domain.map(c => parseConfigValue(c, this.data.conceptProps))
-                : this.data.domain ? this.data.domain
-                : defaults.domain;
+            if (this.config.domain) {
+                return this.config.domain.map(c => parseConfigValue(c, this.data.conceptProps));
+            } else if (this.data.domain) {   
+                //fixBaseline can override the domain if defined and if data domain is one-sided
+                //by replaceing the value closest to zero with the one provided in config
+                //use cases: forcing zero-based bar charts and bubble size
+                if (this.config.fixBaseline != null && isArrayOneSided(this.data.domain)){
```

Comment:
If `baseline <> 0`, `isArrayOneSided` should use that baseline for check right?
And your "closestToZero" doesn't work either for non-0 baseline.
So yeah, let's make it zero for now until we find other use cases. Easier.

Reply:
Should this check if scale is continuous or at least a numeric domain? Is there a use case for zero base line with discrete numeric domains?